### ISSUE TITLE
Website breaks and Stops Working after new connection

### ISSUE THREAD

**Issue by alka1197** [facebookincubator/facebook-for-magento2#18]

**Comment by alka1197** [comment]
Whenever we configured the facebook business extension , website stops working
Magento Version :2.3.1

**Comment by xiangminxufsu** [comment]
Sorry to hear it. Could you attach of a screenshot of logs? It should be located at $MagentoRoot/var/log/facebook-for-magento.log and $MagentoRoot/var/log/debug.log

**Comment by fteixeira777** [comment]
I'm getting the same error! Just homepage works. I have to comment some lines in this files:
Helper\ServerEventFactory.php
Helper\ServerSideHelper.php

Now i can access to categories page with no problems, but some events probably wont work.

If anyone finds a solution please post it.

**Comment by fteixeira777** [comment]
i think the sdk is the problem, because it cant find some classes:
FacebookAds\Api;
FacebookAds\Object\ServerSide\Event;
FacebookAds\Object\ServerSide\EventRequestAsync;

**Comment by xiangminxufsu** [comment]
Yeah our extension depends on facebook/php-business-sdk, so you need to run `composer require facebook/php-business-sdk` as the first step.

**Comment by fteixeira777** [comment]
> Yeah our extension depends on facebook/php-business-sdk, so you need to run `composer require facebook/php-business-sdk` as the first step.

Yes, and i did it!
